lib/torrent.js

```diff
@@ -1579,7 +1584,7 @@ Torrent.prototype._checkDone = function () {
   if (!self.done && done) {
     self.done = true
     self._debug('torrent done: ' + self.infoHash)
-    self.discovery.complete()
+    cb(null)
     self.emit('done')
```

Comment:
Returning true before `_gcSelections` seems to be causing tests to hang. getBuffer never calls its callback: https://github.com/feross/webtorrent/blob/master/test/node/download-dht-magnet.js#L77.  `return self.done` after `_gcSelections` the function resolves this. Thoughts?